samples/tutorial/sym1/build

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+#
+# build.sh
+
+if [ -f $1.c ]; then
+   echo
+   echo "--- Building $1 ---"
+
+   if [ -f $1.s ]; then
+      rm $1.s
+   fi
+   if [ -f $1.o ]; then
+      rm $1.o
+   fi
+   if [ -f $1.bin ]; then
+      rm $1.bin
+   fi
+   if [ -f $1.hex ]; then
+      rm $1.hex
+   fi
+
+   cc65 -t sym1 -O $1.c
+   ca65 $1.s
+   ld65 -C sym1.cfg -m $1.map -o $1.bin $1.o sym1.lib
+   if [ -f $1.bin ]; then
+      bin2hex $1.bin $1.hex > /dev/null 2>&1
+   fi
+   if [ -f $1.hex ]; then
+      echo "--- $1.hex made ---"
+   else
+      echo "--- $1.hex FAIL ---"
+   fi
+
+
+fi
+
```

Comment:
We don't have/want any batch file in this repository, so please remove this file.

In case you want to replace this file with Makefile logic, please be aware that bin2hex is neither part of the cc65 toolchain nor is it listed in https://www.gnu.org/software/make/manual/make.html#Utilities-in-Makefiles so you can't just call it.

Reply:
**REMOVED**